docs/news.rst

```diff
@@ -288,6 +288,8 @@ Backward-incompatible changes
     :class:`~scrapy.http.Request` objects instead of arbitrary Python data
     structures.
 
+*   This version of scrapy breaks custom schedulers 
```

Comment:
I know this is how I phrased it in the issue, but I think the actual documentation text should go more in the line of the entries above.

It should indicate that the `__init__` method of the `scrapy.core.scheduler.Scheduler` class now received an additional `crawler` parameter, and that because of that custom `Scheduler` subclasses that don’t accept arbitrary parameters in their `__init__` method will stop working.

Since we do not have API documentation of `scrapy.core.scheduler.Scheduler` at the moment that we can link, `scrapy.core.scheduler.Scheduler` should probably link to the documentation of the [`SCHEDULER`](https://docs.scrapy.org/en/latest/topics/settings.html#scheduler) setting.

Reply:
@Gallaecio , thanks for the help! Appreciate it!